great_expectations/data_context/cloud_migrator.py

```diff
@@ -299,9 +303,37 @@ def _send_configuration_bundle(
         configuration_bundle: ConfigurationBundle,
         serializer: ConfigurationBundleJsonSerializer,
     ) -> AnyPayload:
-        # Serialize
-        # Use session to send to backend
-        pass
+        url = construct_url(
+            base_url=self._ge_cloud_base_url,
+            organization_id=self._ge_cloud_organization_id,
+            resource_name="migration",
+        )
+
+        serialized_bundle = serializer.serialize(configuration_bundle)
+        data = construct_json_payload(
+            resource_type="migration",
+            organization_id=self._ge_cloud_organization_id,
+            attributes_key="bundle",
+            attributes_value=serialized_bundle,
+        )
+
+        try:
+            response = self._session.post(url, json=data)
+            response.raise_for_status()
+            return response.json()
+
+        except requests.HTTPError as http_exc:
+            raise GeCloudError(
+                f"Unable to migrate config to Cloud: {get_user_friendly_error_message(http_exc)}"
+            )
+        except requests.Timeout as timeout_exc:
+            logger.exception(timeout_exc)
+            raise GeCloudError(
+                "Unable to migrate config to Cloud: This is likely a transient error. Please try again."
+            )
+        except Exception as e:
+            logger.debug(str(e))
```

Comment:
I think we should probably log this at something higher than debug. At least `INFO` or `WARNING`,

Reply:
bumped to warning!